src/routes/legislative-process/+page.svelte

```diff
@@ -45,11 +45,140 @@
 					<h2 class="fluid-heading-04" id="กฎหมายในเว็บไซต์นี้">กฎหมายในเว็บไซต์นี้</h2>
 					<hr />
 				</div>
-				<div>
-					<h2 class="fluid-heading-04" id="ประเภทของกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา">
-						ประเภทของกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา
-					</h2>
-					<hr />
+				<div class="flex flex-col gap-4">
+					<div>
+						<div class="mb-6">
+							<h2 class="fluid-heading-04" id="ประเภทของกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา">
+								ประเภทของกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา
+							</h2>
+							<hr />
+						</div>
+						<p>
+							รัฐสภาประกอบไปด้วยสภาผู้แทนราษฎร และวุฒิสภา
+							ทำหน้าที่พิจารณาและกลั่นกรองร่างกฏหมายก่อนการประกาศใช้ ประกอบด้วย
+						</p>
+					</div>
+					<div class="flex flex-col gap-4">
+						<Collapsible
+							title="ร่างพระราชบัญญัติ (พรบ.)"
```

Comment:
Personally, I think it's looking OK since we only use it on this page, but if you want to extract the data out so that you can use `each of` to loop the component nicely, you can put the data in the load function of `+page.ts` which will be sent to this `+page.svelte` in data prop. Take a look at this as an example #50.

Anyway, both options work fine. Just let me know which way you want to do.

Reply:
i chose the other method. kindly review krub 👍